terraform/aws-creds/.terraform.lock.hcl

```diff
@@ -0,0 +1,22 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Why do we leave this file in the repository? Just to be able to check if there is a `diff` when the file is generated by `terraform init`?

Reply:
It is recommended to add this file to version control, see here: https://www.hashicorp.com/blog/terraform-0-14-introduces-a-dependency-lock-file-for-providers

Reply:
Great, thanks!